plugins/woocommerce/package.json

```diff
@@ -547,6 +550,11 @@
 						"tests/metrics/**",
 						".wp-env.json"
 					],
+					"onlyForDependencies": [
```

Comment:
Changes (Metrics):
- trigger only on intermediate prod-deps changes (other packages should cover each other with plain JS tests and running the job on trunk merge)